### ISSUE TITLE
Atom 1.13 beta 6: tool-bar service is deprecated, needs updating

### ISSUE THREAD

**Issue by SimonAB** [JunoLab/atom-julia-client#286]
On the beta channel of Atom, getting the following message: 

Please update to the latest tool-bar provider service.
```
<unknown> (.atom/packages/tool-bar/lib/tool-bar.js:41:10)
Object.consumeToolBar (.atom/packages/julia-client/lib/package/toolbar.coffee:7:18)
Object.consumeToolBar (.atom/packages/julia-client/lib/julia-client.coffee:68:22)
Provider.provide (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/service-hub/lib/provider.js:29:52)
ServiceHub.provide (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/service-hub/lib/service-hub.js:30:20)
Package.activateServices (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/package.js:363:71)
```

**Comment by waldyrious** [comment]
Why was this closed? I have the same issue in Atom version 1.13.1.

**Comment by pfitzseb** [comment]
Because it's fixed on master (and therefore with the next release).

**Comment by waldyrious** [comment]
Ah, I see, it was fixed on 00d3c1b. Thanks for the clarification.
